### ISSUE TITLE
Adjust path to articles

### ISSUE THREAD

**Issue by larsvilhuber** [amichuda/crress-book#4]
The URL currently is something like
```
https://amichuda.github.io/crress-book/sessions/session1/converted/salmon/salmon.html
```
It should be simplified to 
```
https://amichuda.github.io/crress-book/session1/salmon/index.html
```

**Comment by amichuda** [comment]
That's going to be tough because that will require changing the python code
which creates those folders.

**Comment by larsvilhuber** [comment]
Well, yes, that's the point. Seems pretty straightforward:

https://github.com/amichuda/crress-book/blob/main/src/compile_events.py#L86

and a few other places where all of that is manually defined. It's not deep within Quarto.

**Comment by larsvilhuber** [comment]
One last tweak: could you change `sessions/session5/` to `sessions/5/`?

**Comment by amichuda** [comment]
Closed with 96c59738002ecdb28c7f68d6df1aa49401624da1
